### PR TITLE
wiki: describe Artipie's JFR events

### LINKED ISSUE #1203: Describe Artipie`s JFR events
Create wiki documentation to describe:
- storage events
- slices events

### PR DESCRIPTION
**Pull request by dgarus**
closes #1203 

- added Artipie's JFR events section

### CODE REVIEW

File: .wiki/jfr-events.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+It is integrated into the Java Virtual Machine (JVM) and causes almost no performance overhead, so it can be used 
+even in heavily loaded production environments. When default settings are used, both internal testing and customer 
+feedback indicate that performance impact is less than one percent. For some applications, it can be significantly 
+lower. However, for short-running applications (which are not the kind of applications running in production 
+environments), relative startup and warmup times can be larger, which might impact the performance by more than 
+one percent. JFR collects data about the JVM as well as the Java application running on it.  
+To get more information about JFR, please read [Oracle's Guide](https://docs.oracle.com/javacomponents/jmc-5-4/jfr-runtime-guide/about.htm#JFRUH170).
+
+## Artipie`s Events
+Artipie generate the following events:
```
Comment (olenagerasimova) [inline .wiki/jfr-events.md:13]: generate -> generateS

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, just some cosmetic suggestions

**Comment by dgarus** [comment]
@olenagerasimova thanks! Fixed.

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks

**Pull request merged**